css/style.css

```diff
@@ -1,11 +1,32 @@
-.card {
-    margin: 40px 10% 20px; 
+.container {
+  display: grid;
+  margin: 0 auto;
+  grid-gap: 2px;
+  background-color: azure;
 }
-section {
-    padding-top: 0px;  
-    margin-top: 0px;  
-    display: block;  
-    height: 100vh;
+
+header {
+  display: block;
+  padding: 5px;
+}
+
+.wrap {
+  display: block;
+  margin: 2px 6px auto;
 }
```

Comment:
Please use code that is related to your webpage. The .wrap class has not been used in any of your pages